Assignment 1.R

```diff
@@ -0,0 +1,35 @@
+
+# Problem 2
+
+# creating column names for col.names argument in read_csv
+column_names <- c("name", "a_26", "m_b", "log_lk", "log_m26", "log_mhi", "vlg", "ti1", "md", "D", "delta_vlg", "count")
+
+
+# Read the text file using read.csv
+galaxies <- read.csv("suites_dw_Table1.txt", header = FALSE, sep ="|",  skip = 16, col.names = column_names,
+                     strip.white = TRUE)
+
+
+# View the first rows of the data
+head(galaxies)
+
+# Summary statistics 
+summary(galaxies)
+
+
+# Problem 3
+
+#import for plotting 
+library(ggplot2)
+
+# Create a histogram for linear diameter

```

Comment:
A line plot with distance vs size would have shown that there are more observed galaxies close to us that far away from us. This may be an indicator that our technology doesnt capture small galaxies far away from us.